src/Behat/Testwork/ServiceContainer/Configuration/ConfigurationLoader.php

```diff
@@ -195,7 +197,20 @@ protected function loadFileConfiguration($configPath, $profile)
         }
 
         $basePath = rtrim(dirname($configPath), DIRECTORY_SEPARATOR);
-        $config = (array) Yaml::parse(file_get_contents($configPath));
+
+        if (
+            \str_ends_with($configPath, '.php')
+            || \str_ends_with($configPath, '.php.dist')
```

Comment:
I'd prefer a `use str_ends_with;` rather than the inline `\`

Reply:
also done